bench/decode.ts

```diff
@@ -0,0 +1,88 @@
+import {
+  ConversationV1,
+  ConversationV2,
+} from './../src/conversations/Conversation'
+import { MessageV1 } from '../src/Message'
+import { newLocalHostClient } from '../test/helpers'
+import { utils } from '../src/crypto'
+import {
+  MESSAGE_SIZES,
+  newPrivateKeyBundle,
+  randomBytes,
+  wrapSuite,
+} from './helpers'
+import { add } from 'benny'
+import { fetcher } from '@xmtp/proto'
+
+const decodeV1 = () => {
+  return MESSAGE_SIZES.map((size) =>
+    add(`decode and decrypt a ${size} byte v1 message`, async () => {
+      const alice = await newLocalHostClient()
+      const bob = await newPrivateKeyBundle()
```

Comment:
The benchmark includes setup cost such as creating a new private key bundle, etc right?

Reply:
No. The setup happens before the benchmark timer begins. Only the returned function is part of the actual benchmark.

Reply:
oh neat!